packages/legend-studio/src/components/editor/edit-panel/mapping-editor/InstanceSetImplementationEditor.tsx

```diff
@@ -443,6 +443,11 @@ export const InstanceSetImplementationEditor = observer(
         PureInstanceSetImplementationState &&
       instanceSetImplementationState.mappingElement.filter;
 
+    const mappingElementDecorationCleaner = useMemo(() => {
```

Comment:
not sure why we need to this? why don't we just put this directly in the `useEffect` instead of using `useMemo` like this here?